python/cudf/cudf/core/series.py

```diff
@@ -4664,6 +4664,41 @@ def floor(self, freq):
             data={self.series.name: out_column}, index=self.series._index
         )
 
+    def round(self, freq):
+        """
+        Perform round operation on the data to the specified freq.
+
+        Parameters
+        ----------
+        freq : str
+            One of ["D", "H", "T", "min", "S", "L", "ms", "U", "us", "N"].
+            Must be a fixed frequency like 'S' (second) not 'ME' (month end).
+            See `frequency aliases <https://pandas.pydata.org/docs/\
+                user_guide/timeseries.html#timeseries-offset-aliases>`__
+            for more details on these aliases.
+
+        Returns
+        -------
+        Series
+            Series with all timestamps rounded up to the specified frequency.
+            The index is preserved.
+
+        Examples
+        --------
+        >>> import cudf, pandas
+        >>> rng = pandas.date_range('1/1/2018 11:59:00', periods=3, freq='min')
+        >>> cudf.Series(rng).dt.round("H")
+        0   2018-01-01 12:00:00
+        1   2018-01-01 12:00:00
+        2   2018-01-01 12:00:00
```

Comment:
This feels like an unhelpful example. Could we construct an example that has unique results? Perhaps copy from another function's docstring.

Reply:
As a very minor suggestion, I would prefer a different variable name like `dates` instead of `rng`. The word `rng` means "random number generator" to me, not "range."